### PR TITLE
[UI/FIX] Fix Library header overlaps search suggestions

### LINKED ISSUE #2561: Library header overlaps search suggestions
### Describe the bug

image

### Add logs

```text
not relevant
```

### Steps to reproduce

1. Start searching something

### Expected behavior

The suggestions should be OVER the header, not below.

### Screenshots

_No response_

### Heroic Version

main branch from GitHub

### System Information

not relefant

### Additional information

this is a regression in the main branch, not present in stable

### PR DESCRIPTION
**Pull request by Etaash-mathamsetty**
fixes #2561

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: public/locales/en/login.json
Comment (CommandMC) [inline public/locales/en/login.json:1]: These changes seem unrelated

Comment (Etaash-mathamsetty) [reply public/locales/en/login.json:1]: yeah they show up when I push for some reason, I will try to remove them

Comment (arielj) [reply public/locales/en/login.json:1]: this should be fine, it shows up when we run `yarn i18next` in the main branch so we will need this anyway, maybe something change in the update of dependencies and now it's sorting those keys in alphabetical order

File: src/frontend/components/UI/SearchBar/index.css [hunk 1]
```diff
@@ -25,6 +25,7 @@
   padding: var(--space-xs) var(--space-md);
   text-align: left;
   overflow-x: hidden;
+  z-index: 1;
```
Comment (arielj) [inline src/frontend/components/UI/SearchBar/index.css:28]: this z-index is not really needed, it's enough with the z-index: 10 in the Header css

Comment (Etaash-mathamsetty) [reply src/frontend/components/UI/SearchBar/index.css:28]: Nope this is needed as well, otherwise the filters go on top of the search suggestions

**Comment by Etaash-mathamsetty** [comment]
screenshot: (after, before is in the original issue)
image

**Review by arielj — APPROVED** [review]
Looks good.

I think at some point we'll have to refactor all this `z-index` values, now it's hard to maintain or know what goes over what, but that would be another PR
